package/src/components/Message/Message.tsx

```diff
@@ -598,6 +601,7 @@ const MessageWithContext = <
       onPressProp
```

Comment:
This is a very unreadable part, can we improve this onPress and extract it to a func. at the top here? 
The number of inline ternary operations is a no go.

I know you're just adding a small part, but let's make the codebase even more readable and maintainable!

Reply:
```suggestion
      const onPressArgs ={
          actionHandlers,
          additionalInfo: payload.additionalInfo,
          defaultHandler: payload.defaultHandler || onPress,
          emitter: payload.emitter || 'message',
          event: payload.event,
          message,
        }
          
      const handleOnPress = ()=> {
          if(onPressProp) return onPressProp(onPressArgs)
          if(onPressMessageProp) return onPressMessageProp(onPressArgs)
          if(payload.defaultHandler) return payload.defaultHandler()
          
          return onPress()             
      }
```

This is so much more readable, please test carefully!

Reply:
✅